aeson.cabal

```diff
@@ -162,6 +162,9 @@ test-suite tests
     vector,
     quickcheck-instances >=0.3.12
 
+  if !impl(ghc >= 8.0)
+    build-depends: semigroups >= 0.16.1
```

Comment:
@hvr https://github.com/bos/aeson/commit/2027fb84c60c08af70fbed1da7f88ee9d9a9cfb8 You are the best one to answer? :)